src/stepfunctions/steps/sagemaker.py

```diff
@@ -64,7 +68,7 @@ def __init__(self, state_id, estimator, job_name, data=None, hyperparameters=Non
                 * (list[sagemaker.amazon.amazon_estimator.RecordSet]) - A list of
                     :class:`sagemaker.amazon.amazon_estimator.RecordSet` objects,
                     where each instance is a different channel of training data.
-            hyperparameters (dict, optional): Specify the hyper parameters for the training. (Default: None)
+            hyperparameters (dict, optional): Specify the hyperparameters that are set before the model begins training. If hyperparameters provided are also specified in the estimator, the provided value will used. (Default: Hyperparameters specified in the estimator will be used for training.)
```

Comment:
> If hyperparameters provided are also specified in the estimator, the provided value will used.

The code comment you wrote about merging was a clearer description. I don't think this sentence communicates that the `hyperparameter` arg values are merged into the estimator's hyperparameters.


---

As for other documentation, I think the docstring suffices since that shows up in RTD.

Reply:
gotcha, i'd taken 2 stabs and was waffling back and forth on which one I understood better. will go with the other blurb